src/device-registry/models/Site.js

```diff
@@ -13,6 +13,16 @@ const siteSchema = new Schema(
       trim: true,
       required: [true, "name is required!"],
     },
+    share_links: {
+      preview: { type: String, trim: true },
+      share: { type: String, trim: true },
```

Comment:
Hi @Baalmart I think let's call this `short_link` because we have a long link and a short link

Reply:
Alright @NoahNsimbe , let me make the update asap...

Reply:
DONE. @NoahNsimbe

Reply:
oh... sorry I meant changing `share` to `short_link` i.e
```
share_links: {
      preview: { type: String, trim: true },
      short_link: { type: String, trim: true },
}
```
@Baalmart

Reply:
Hi @NoahNsimbe , please check out the recently pushed changes.
<img width="813" alt="Screenshot 2023-01-08 at 12 26 38" src="https://user-images.githubusercontent.com/1590213/211256911-4af16270-04e1-4133-b016-9cb0d357b004.png">